solr/contrib/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedFeatureStore.java

```diff
@@ -174,10 +174,10 @@ public void doGet(BaseSolrResource endpoint, String childId) {
   }
 
   private static List<Object> featuresAsManagedResources(FeatureStore store) {
-    final List<Object> features = new ArrayList<Object>(store.size());
+    final List<Object> features = new ArrayList<Object>();
     for (final Feature f : store.getFeatures()) {
       final LinkedHashMap<String,Object> m = toFeatureMap(f);
-      m.put(FEATURE_STORE_NAME_KEY, store.getName());
+      m.put(STORE_KEY, store.getName());
```

Comment:
'feature' is already implied by the class name, though actually this change is unrelated to making models pluggable and so I will revert it for clarity.